files/usr/share/cinnamon/applets/grouped-window-list@cinnamon.org/appGroup.js

```diff
@@ -466,16 +472,14 @@ class AppGroup {
         if (this.label.text == null) {
             this.label.set_text('');
         }
-        // TODO: This should be set by the theme.
-        this.label.set_style('padding-right: 4px;');
 
         if (!animate) {
             this.label.show();
-            this.label.width = width;
+            this.actor.set_width(width);
```

Comment:
Minor nitpick - in GI these properties have setters which call the same functions - I think its cleaner and more composable as prop assignment.